<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>881</width>
    <height>728</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="cash_box">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>20</y>
     <width>111</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>지폐투입구</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="change_box">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>200</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>거스름돈</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QPushButton" name="change_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>88</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>20</y>
     <width>111</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>카드투입구</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QPushButton" name="card_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>88</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>투입</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="receipt_box">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>200</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>영수증 나오는 곳</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QPushButton" name="receipt_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>88</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>620</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>긴급상황</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>36</y>
      <width>88</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>CALL</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="products">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>501</width>
     <height>511</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>상품선택</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="recommend">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>20</y>
     <width>111</width>
     <height>271</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>추천해주세요!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>530</y>
     <width>501</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>음료 나오는 곳</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QPushButton" name="drink_btn">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>130</y>
      <width>88</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>받았습니다!</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="school_label">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>310</y>
     <width>331</width>
     <height>301</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
